backend/de.metas.manufacturing/src/main/java/de/metas/manufacturing/generatedcomponents/ComponentGeneratorParam.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * #%L
+ * de.metas.manufacturing
+ * %%
+ * Copyright (C) 2020 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.manufacturing.generatedcomponents;
+
+import com.google.common.collect.ImmutableMap;
+import lombok.Data;
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+@Data(staticConstructor = "of")
+public class ComponentGeneratorParam
```

Comment:
i think `ComponentGeneratorParams` (plural) would be more suggestive.

Reply:
makes sense.

Done